catalog/tests/test_views.py

```diff
@@ -52,13 +52,16 @@ def test_lists_all_authors(self):
 from catalog.models import BookInstance, Book, Genre, Language
 from django.contrib.auth.models import User  # Required to assign User as a borrower
 
+# Get user model from settings
+from django.contrib.auth import get_user_model
+UserModel = get_user_model()

```

Comment:
It would be more convenient to name the model **User** instead of **UserModel** for consistency. 
User = get_user_model()
```suggestion
User = get_user_model()
```